Source/UI/NewUI/BillingForm/View/MainBillingFormTextField.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+final class MainBillingFormTextField: UITextField, BillingFormTextField {
```

Comment:
Should `BillingFormTextField` inherit from `UITextField`?

Reply:
Question : what is MainBillingFormTextField ? Main refers to here ? and What is the difference between MainBillingFormTextField and BillingFormTextField

Reply:
I tried but.i got this `'BillingFormTextField' requires that 'DefaultBillingFormTextField' inherit from 'UITextField`
`MainBillingFormTextField` is renamed to `DefaultBillingFormTextField`